cli/lib/appdata.ts

```diff
@@ -163,20 +173,29 @@ export function getNewSitePartial( siteFolder: string ): z.infer< typeof siteSch
 	return newSite;
 }
 
-export async function getOrCreateSiteByFolder(
+const createNewSite = async ( siteFolder: string ): Promise< z.infer< typeof siteSchema > > => {
```

Comment:
```suggestion
const createNewSite = async ( siteFolder: string ): Promise< SiteData > => {
```
Nit, but there's a dedicated type for this now.